foundation-extensions/places/places-usage-reference.md

```diff
@@ -279,6 +280,47 @@ MobileCore.registerExtensions([Places.self])
 {% endtab %}
 {% endtabs %}
 
+### setAccuracyAuthorization
+
+Sets the accuracy authorization status in the Places extension.
+
+The value provided is stored in the Places shared state, and is for reference only. Calling this method does not impact the actual location accuracy authorization for this device.
+
+{% tabs %}
+
+{% tab title="Swift"}
+
+**Signature**
```

Comment:
We usually use Syntax and Example instead of Signature and Example Usage. Should we update all to Syntax and Example for consistency?

Reply:
Yeah, that's one of the things I'm big on - making sure we're consistent throughout all our docs for what we stub things as.

I don't have any preferences for if we choose Signature/Example Usage or Syntax/Example, but the important thing is that we're consistent across all our docs that we write.